generators/spa/templates/VueJS/common/View.html

```diff
@@ -16,29 +19,31 @@ <h3>Item list </h3>
         <table class="table table-striped">
             <thead>
                 <tr>
-                    <th>Id</th>
-                    <th>Name</th>
-                    <th>Description</th>
+                    <th v-if="settings.itemId">Id</th>
+                    <th v-if="settings.name">Name</th>
+                    <th v-if="settings.description">Description</th>
+                    <th v-if="settings.createdOnDate">Created on</th>
                     <th></th>
                 </tr>
             </thead>
             <tbody>
                 <tr class="tm_t" v-for="(item, index) in items" :key="item.id">
-                    <td>{{item.ItemId}}</td>
-                    <td>{{item.ItemName}}</td>
-                    <td class="tm_td">{{item.ItemDescription}}</td>
+                    <td v-if="settings.itemId">{{item.id}}</td>
+                    <td v-if="settings.name">{{item.name}}</td>
+                    <td v-if="settings.description">{{item.description}}</td>
+                    <td v-if="settings.createdOnDate">{{item.createdOnDate}}</td>
                     <td>
                         <button v-if="userCanAdd" type="button" class="btn btn-sm" title="Edit item" @click="editItem(item)">
                             <i class="glyphicon glyphicon-edit"></i>
                         </button>
-                        <button v-if="userCanAdd" type="button" class="btn btn-sm btn-danger" title="Remove item" @click="deleteItem(item.ItemId)">
+                        <button v-if="userCanAdd" type="button" class="btn btn-sm btn-danger" title="Remove item" @click="deleteItem(item.id)">
```

Comment:
@VAlmea Would `Id` or `ItemId` be a better object name?

Reply:
@VAlmea ???

Reply:
Fixed